docs/components/slider.md

```diff
@@ -0,0 +1,220 @@
+# Slider
+
+KSlider is a draggable horizontal input for selecting a numeric value within a range.
+
+<KSlider
+  v-model="vModel1"
+  label="How many nodes do you want to run in this cluster?"
+  :min="1"
+/>
+
+```html
+<KSlider
+  v-model="nodesCount"
+  label="How many nodes do you want to run in this cluster?"
+  :min="1"
+/>
+```
+
+## Props
+
+### v-model
+
+KSlider works with `v-model` for two-way data binding.
+
+<KSlider
+  v-model="vModel2"
+  label="How many nodes do you want to run in this cluster?"
+  :min="1"
+/>
+
+```vue
+<template>
+  <KSlider
+    v-model="nodesCount"
+    label="How many nodes do you want to run in this cluster?"
+    :min="1"
+  />
+</template>
+
+<script setup lang="ts">
+const nodesCount = ref<number>(5)
+</script>
+```
+
+### label
+
+String to be used as label. Default value is empty string (`''`).
+
+<KSlider
+  v-model="vModel3"
+  label="In the binary system, which value is used to represent true?"
+  :max="1"
+/>
+
+```html
+<KSlider
+  label="In the binary system, which value is used to represent true?"
+  v-model="binaryTrue"
+  :max="1"
+/>
+```
+
+### min
+
+Minimum value of the range. Default value is `0`.
+
+### max
+
+Minimum value of the range. Default value is `10`.
+
+### step
+
+Step value. Default value is `1`.
+
+<KSlider
+  v-model="vModel4"
+  :max="100"
+  :min="40"
+  show-marks
+  :step="5"
+/>
+
+```html
+<KSlider
+  v-model="sliderValue"
+  :max="100"
+  :min="40"
+  :step="5"
+  show-marks
+/>
+```
+
+### showMarks
+
+Boolean value to control whether or not marks for every step on the range should be shown. Marks for minimum and maximum values are shown at all times.
+
+Defaults to `false`.
+
+<KSlider
+  v-model="vModel5"
+  :max="50"
+  show-marks
+  :step="5"
+/>
+
+```html
+<KSlider
+  show-marks
+  v-model="sliderValue"
+  :max="50"
+  :step="5"
+/>
+```
+
+### marks
```

Comment:
IMO "marks" is a bit confusing; there's not an inherit relationship to "steps" here

Reply:
Initially Max named it as `labels` but we both agree it may cause confusion as we already have `label`.

Reply:
They are called mark texts according to [Open UI](https://open-ui.org/components/slider.research.parts/). But we do not have those little ticks on the track (which I believe are the "marks") so marks can be confusing as well…

Reply:
You could call it `markLabels` or `tickLabels`. `tick` may be a little more well known than `mark`?

Reply:
> I suggest we leave it as is as all other options feel same as this one - "a little confusing / unintuitive" 😅

I'm not sure I agree. 

This needs to imply "available option labels" 

I think the current implementation is a bit confusing, especially considering some of the issues in identifying the selected value I commented separately.